src/components/EnergyConsumptionChart.tsx

```diff
@@ -1,241 +1,264 @@
-        color: "#F2A900",
-      },
-      {
-        name: t("energy-consumption-chart-gaming-us-label"),
-        amount: 34,
-        color: "#71BB8A",
-      },
-      {
-        name: t("energy-consumption-chart-eth-pow-label"),
-        amount: 21,
-        color: "#C1B6F5",
-      },
-      {
-        name: t("energy-consumption-chart-eth-pos-label"),
-        amount: 0.0026,
-        color: "#C1B6F5",
-      },
-    ],
-    sm: [
-      {
-        name: t("energy-consumption-chart-global-data-centers-label"),
-        amount: 190,
-        color: "#FF0000",
-      },
-      {
-        name: t("energy-consumption-chart-btc-pow-label"),
-        amount: 149,
-        color: "#D7B14A",
-      },
-      {
-        name: t("energy-consumption-gold-mining-cbeci-label"),
-        amount: 131,
-        color: "#F2A900",
-      },
-      {
-        name: t("energy-consumption-chart-eth-pow-label"),
-        amount: 21,
-        color: "#C1B6F5",
-      },
-      {
-        name: t("energy-consumption-chart-netflix-label"),
-        amount: 0.457,
-        color: "#E50914",
-      },
-      {
-        name: t("energy-consumption-chart-eth-pos-label"),
-        amount: 0.0026,
-        color: "#C1B6F5",
-      },
-    ],
-    md: [
-      {
-        name: t("energy-consumption-chart-global-data-centers-label"),
-        amount: 190,
-        color: "#FF0000",
-      },
-      {
-        name: t("energy-consumption-chart-btc-pow-label"),
-        amount: 149,
-        color: "#D7B14A",
+  // chart labels
+  const labels = [
+    t("energy-consumption-chart-global-data-centers-label"),
+    t("energy-consumption-chart-btc-pow-label"),
+    t("energy-consumption-gold-mining-cbeci-label"),
+    t("energy-consumption-chart-gaming-us-label"),
+    t("energy-consumption-chart-eth-pow-label"),
+    "Google",
+    t("energy-consumption-chart-netflix-label"),
+    t("energy-consumption-chart-paypal-label"),
+    t("energy-consumption-chart-airbnb-label"),
+    t("energy-consumption-chart-eth-pos-label"),
+  ]
+
+  const chartOptions = {
+    // chart styles
+    barThickness: 38,
+    borderRadius: 4,
+    aspectRatio: 1.1,
+    responsive: true,
+    maintainAspectRatio: true,
+    hover: { mode: null } as any, // casted to avoid TS issue
+    backgroundColor: ENERGY_CONSUMPTION_CHART_COLORS as any, // casted to avoid TS issue
+    plugins: {
+      // required plugin to display labels on top
+      datalabels: {
+        // https://chartjs-plugin-datalabels.netlify.app/guide/positioning.html#alignment-and-offset
+        anchor: "end" as any, // position of the labels (start, end, center), casted to avoid TS issue
+        align: "end" as any, // alignment of the labels (start, end, center), casted to avoid TS issue
+        offset: -0.5, // distance (in pixels) to pull the label away from the anchor point
+        font: {
+          size: "14px",
+        } as any, // casted to avoid TS issue,
+        color: useColorModeValue("#333333", "#F2F2F2"),
```

Comment:
Could we use theme tokens here? (and throughout for font sizes / colors, etc)

Reply:
Doesn't work with the tokens